kubernetes/resource_kubernetes_persistent_volume_claim_test.go

```diff
@@ -172,7 +170,7 @@ func TestAccKubernetesPersistentVolumeClaim_googleCloud_volumeMatch(t *testing.T
 				),
 			},
 			{
-				ResourceName:            resourceName,
+				ResourceName:            "kubernetes_persistent_volume_claim",
```

Comment:
This change wasn't necessary, but it was for convenience and consistency during rebase.